tests/Http/Api/PostApiTest.php

```diff
@@ -1,57 +1,126 @@
 <?php
 
+use Rogue\Models\Signup;
+use Illuminate\Support\Facades\Storage;
+
 class PostApiTest extends TestCase
 {
     /**
-     * Test that a POST request to /posts creates a new photo post.
+     * Test that a POST request to /posts creates a new
+     * photo post and signup, if it doesn't already exist.
```

Comment:
I debate whether we should, in fact, create a signup on the fly if there isn't one or if we should not allow creating posts if there isn't a signup already. Perhaps, something to think about for the future.